Expensify/App
# Show users in from your private domain in search even if you don't have an existing conversation

### Details
1. This shows users in your domain (anyone you have personalDetails for) in the search bar even if you don't have a conversation with them
2. The search page now creates conversations when you select someone you don't have a conversation with
3. If you typed a name that doesn't exist into search and pressed enter, the app crashes. It doesn't do that now

### Fixed Issues
$ https://github.com/Expensify/Expensify/issues/153863

### Tests
1. Go to web
2. Press `CMD+K`
2. Type `asodifjasodifjaosifjaoisfj` into the search and press enter
3. Make sure the app doesn't crash
4. Type the email or name of someone on your domain that you don't have an existing conversation with and make sure their name shows up
5. Click their name and make sure a conversation is started

### Tested On
